build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Use this with `npm run build` so that `./node_modules/.bin` are injected into
+# the PATH
+
+license='/*! algoliasearch 2.9.2 | © 2014, 2015 Algolia SAS | github.com/algolia/algoliasearch-client-js */'
```

Comment:
Yes I agree, after doing this build script, I thought the same. I hate it.

Maybe I'll go gulp/shelljs or env variable injection, I am sure a module exists, will have a look.